pontoon/base/models.py

```diff
@@ -180,6 +183,28 @@ def user_translated_locales(self):
 
     return [locale.code for locale in locales]
 
+
+@property
+def user_rights(self):
+    output = 'Contributor'
+
+    if self.translated_locales:
+        if self.is_superuser:
+            locales = 'all locales'
+        else:
+            locales = ', '.join(self.translated_locales)
+
+        locale = get_object_or_404(Locale, code__iexact=self.translated_locales[0])
```

Comment:
We actually need to create `managed_locales` and check `if self.managed_locales:` and then list them. Currently you only check if the user is a manager of the first locale he is a translator for and then list all translated_locales as locales he is a manager for.